course_content/iris_course/4.Joining_Cubes_Together.ipynb

```diff
@@ -55,69 +57,255 @@
    "source": [
-    "Any CubeList can be merged with the ``merge`` method, and the resulting CubeList from load_raw is no different.\n",
-    "The ``merge`` method *always* returns another CubeList:"
+    "To merge a CubeList, we can use the ``merge`` or ``merge_cube`` methods. \n",
+    "\n",
+    "The `merge` method will try to merge together the cubes in the CubeList in order to return a CubeList of as few cubes as possible.\n",
+    "\n",
+    "The `merge_cube` method will do the same as `merge` but will return a single Cube. If the initial CubeList cannot be merged into a single Cube, `merge_cube` will raise an error, giving a helpful message explaining why the cubes cannot be merged.\n",
+    "\n",
+    "Let's merge the raw 2D cubes we previously loaded in:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 6,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0: surface_temperature / (K)           (time: 6; latitude: 145; longitude: 192)\n"
+     ]
+    }
+   ],
    "source": [
-    "merged_cube, = raw_cubes.merge()\n",
+    "merged_cubelist = raw_cubes.merge()\n",
+    "print(merged_cubelist)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`merge` has returned a cubelist of a single 3D cube."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "surface_temperature / (K)           (time: 6; latitude: 145; longitude: 192)\n",
+      "     Dimension coordinates:\n",
+      "          time                           x            -               -\n",
+      "          latitude                       -            x               -\n",
+      "          longitude                      -            -               x\n",
+      "     Auxiliary coordinates:\n",
+      "          forecast_period                x            -               -\n",
+      "     Scalar coordinates:\n",
+      "          forecast_reference_time: 2011-07-22 00:00:00\n",
+      "          realization: 8\n",
+      "     Attributes:\n",
+      "          STASH: m01s00i024\n",
+      "          source: Data from Met Office Unified Model\n",
+      "          um_version: 7.6\n",
+      "     Cell methods:\n",
+      "          mean: time (1 hour)\n"
+     ]
+    }
+   ],
+   "source": [
+    "merged_cube = merged_cubelist[0]\n",
     "print(merged_cube)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When we look in more detail, we can see that the time coordinate has become a new dimension, as well as gaining another forecast_period auxiliary coordinate:"
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "    <b><font color='brown'>Exercise: </font></b>Try merging `raw_cubes` using  the `merge_cube` method.\n",
```

Comment:
Note I have not given a solution to these exercises as they are very trivial!

Reply:
:+1: